.github/scripts/webhook-pr-comment.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const path = require('path');
+module.exports = () => {
```

Comment:
I created this script because I couldn't figure out from the documentation how to get properties with dashes using the `fromJson`. Also, the property that holds the webhook URL is also an object that contains an arbitrary key, so using the script made it way easier to pluck it out.